cmd/remotefs/main.go

```diff
@@ -54,7 +54,7 @@ func usage() {
 
 func main() {
 	base64string := flag.String("base64", "", "base64-encoded json string with all information")
-	logLevel := flag.String("loglevel", "warning", "Logging Level: trace, debug, info, warning, error, fatal, panic.")
+	logLevel := flag.String("loglevel", "debug", "Logging Level: trace, debug, info, warning, error, fatal, panic.")

```

Comment:
We would still want the default log level to be warning or error.  The defaults should be safe to use.  Debug can leak secrets which we wouldn't want by default.

Reply:
done